scripts/forms-validation.js.php

```diff
@@ -1,18 +0,0 @@
-<?php
-include ('../../../inc/includes.php');
-header('Content-type: text/javascript');
-?>
-
-function validateForm(form) {
```

Comment:
are you sure about validateForm removal? 

cf: https://github.com/pluginsGLPI/formcreator/blob/master/inc/form.class.php#L838